Examples/Run/Common/include/ActsExamples/Seeding/SeedingExampleBase.hpp

```diff
@@ -0,0 +1,22 @@
+// This file is part of the Acts project.
```

Comment:
Will it be better to create a directory `Examples/Run/Seeding` and move those files there?

Reply:
Actually I used the same structure as the Digitization example. If you think Example/Run/Seeding is better, I can move those file there.

Reply:
OK, I moved those files in Example/Run/Seeding